openedx_certificates/renderers/elements.py

```diff
@@ -139,3 +139,13 @@ def draw_text(certificateGen, attributes, canvas, context):
 
 def draw_template_element(certificateGen, element, attributes, canvas, context=None):
     ELEMENT_OPTIONS[element](certificateGen, attributes, canvas, context)
+
+
+def draw_flair(certificateGen, flair, order, page, context=None):
```

Comment:
Since we always pass in `context`, we shouldn't be assigning a default value here.